src/Microsoft.Framework.PackageManager/SourceControl/SourceCommand.cs

```diff
@@ -0,0 +1,244 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Framework.Runtime;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NuGet;
+
+namespace Microsoft.Framework.PackageManager.SourceControl
+{
+    internal class SourceCommand
+    {
+        private const string SourcesFolderName = "sources";
+
+        private const string SourcesEnvironmentVariable = "DNX_SOURCES_DIR";
```

Comment:
We have `DNX_PACKAGES` instead of `DNX_PACKAGES_DIR`. Let's follow the same naming convention.